etl/steps/data/garden/homicide/2024-10-30/unodc.meta.yml

```diff
@@ -0,0 +1,88 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Homicides
+
+  metric:
+    <% if unit_of_measurement == "Counts" %>
+    Number of homicides
+    <% elif unit_of_measurement == "Rate per 100,000 population" %>
+    Homicide rate per 100,000 population
+    <%- endif -%>
+  sex:
+    <% if sex == "Total" %> all victimes<% elif sex == "Male" %>male victims<% elif sex == "Female" %>female victims<%- endif -%>
+  age:
```

Comment:
nitpick but: why do some age groups start with "aged..." and some don't?

Reply:
Ooops yep, that's a mistake!